client/src/components/ImageGallery/Swiper.scss

```diff
@@ -84,14 +84,26 @@
 }
 
 
-@media screen and (max-width: 1100px){
+@media screen and (max-width: 1220px){
     .c{
         margin: 20px;
+    };
+    .swiper{
+        max-width: 1000px;
+    }
+}
+
+@media screen and (max-width: 820px){
+    .c{
+        margin: 20px;
+    };
+    .swiper{
+        max-width: 700px;
     }
 }
 
 
-@media screen and (max-width: 480px){
+@media screen and (max-width: 500px){
     .c{
```

Comment:
please check the breakpoints in our theme.js file. 

also i could suggest using the Container component since by default it is already adjusted to our breakpoints,  we are using it across all our project to wrap our content in order to have uniform margins and paddings.  all already included in our created theme.

Reply:
issue not addressed in this merge. postponed to later